hubverse_annotator/app.py

```diff
@@ -232,19 +232,51 @@ def reference_date_and_location_ui(
     """
     loc_lookup = get_available_locations(observed_data_table, forecast_table)
     long_names = loc_lookup["long_name"].to_list()
+    if "locations_list" not in st.session_state:
+        st.session_state.locations_list = sorted(long_names)
+    if "location_selection" not in st.session_state:
+        st.session_state.location_selection = long_names[0]
+
+    def prev_loc():
+        loc = st.session_state.locations_list.index(
+            st.session_state.location_selection
+        )
+        st.session_state.location_selection = st.session_state.locations_list[
+            loc - 1
+        ]
+
+    def next_loc():
+        loc = st.session_state.locations_list.index(
+            st.session_state.location_selection
+        )
+        st.session_state.location_selection = st.session_state.locations_list[
+            (loc + 1) % len(st.session_state.locations_list)
```

Comment:
I think this implements a "wrap around," but I'd rather have it stop, so it is obvious when you have gone through all the locations.